components/Container.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactChildren } from "react";
+
+type ContainerType = {
+    children: JSX.Element|JSX.Element[];
+}
+
+const Container = ({ children }: ContainerType) => {
+    return (
+        <main className=" max-w-screen-md my-10 px-5 container mx-auto">
+            {children}
```

Comment:
Indentation is not consistent between files, try to choose one and stick to it 😉